test/e2e/tests/send-eth.spec.js

```diff
@@ -251,25 +251,13 @@ describe('Send ETH from dapp using advanced gas controls', function () {
         );
         await driver.assertElementNotPresent({ text: 'Data', tag: 'li' });
         await driver.clickElement({ text: 'Edit', tag: 'button' });
-        await driver.waitForSelector({
```

Comment:
should we keep one of these `.transaction-total-banner` checks to verify the value still exists?

Reply:
done